jwala-services/src/main/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImpl.java

```diff
@@ -76,6 +83,13 @@ public WebServer createWebServer(final CreateWebServerRequest createWebServerReq
                                      final User aCreatingUser) {
         createWebServerRequest.validate();
 
+        List<Jvm> jvmList = jvmPersistenceService.getJvms();
+        for (Jvm jvm : jvmList) {
+            if(createWebServerRequest.getName().equals(jvm.getJvmName())){
+                throw new EntityExistsException("Jvm already exists with this name "+ createWebServerRequest.getName());
```

Comment:
Like Jedd stated earlier, throw a WebServerServiceException here. I'm guilty of this as well, but now I know what to look for in the future.